tools/collect_debug_info.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+#
+# This script will save debug info to either a gist or /tmp/
+
+# Are we running as root?
+if [[ $EUID -ne 0 ]]; then
+  echo "This script must be run as root. Please re-run like this:"
+  echo
+  echo "sudo $0"
+  echo
+  exit
+fi
+
+echo "This script produces a diagnostic log to help the maintainers"
+echo "figure out why your Mail-in-a-Box installation isn't working the"
+echo "way you expected it to."
+
+echo
+echo "This log will contain sensitive information about your installation"
+echo "including, but not limited to:"
+echo "email addresses"
+echo "domain names"
+echo "IP addresses"
+echo "server security configuration"
+echo "etc."
+echo
+echo "====================================================================="
+echo "Please do not post this file to the internet if you are not comfortable"
+echo "exposing this information publicly to the world forever."
+echo "====================================================================="
+echo
+echo "Once the log has been collected, you will be given the option to post"
+echo "the log to https://gist.github.com/ so that others can help you diagnose"
+echo "the issues with your Mail-in-a-Box installation"
+echo
```

Comment:
Perhaps we can add a message that they can filter the log themselves and remove sensitive information and then manually post it? 

Perhaps we should also mention it requires a GitHub account to be able to post.